packages/shared/src/styles/themes.scss

```diff
@@ -23,6 +23,11 @@
     --text-weight-bold: bold;
     --text-weight-bolder: bolder;
 
+    //Text Align
+    --text-align-start: start;
```

Comment:
Can we stick to `left`?

Reply:
it has been considered the languages like Farsi..